kippenhahn/kippenhahn.py

```diff
@@ -1345,14 +1371,15 @@ def plot(self, ax=None):
 
 
         #Re-enforcing the calculated limits for X and Y axis. Without this there may be a white band on the right of the plot.
-        ax1.set_xlim([self._Xaxis_min,self._Xaxis_max])
-        ax1.set_ylim([self._Yaxis_min,self._Yaxis_max])
-        if self._param['abundances']:
-            ax2.set_xlim([self._Xaxis_min,self._Xaxis_max])
-            if self._param['log_abundances']:
-                ax2.set_ylim([1e-5,1.])
-            else:
-                ax2.set_ylim([0.,1.])
+        if ax is not None:
```

Comment:
Are you sure you don't want to check for `self._param['ax']` here?
Even a deeper thought: shouldn't we check, whether `self._Xaxis_min`, `self._Xaxis_max`, ... are given? Usually, I'd let the user overwrite old limits, when new ones are given.